wordpress-mobile/WordPress-Android
# Improve horizontal scroll behaviour

This PR improves the horizontal scroll behaviour. Previously if you didn't swipe exactly horizontally, the scroll would be interpreted as vertical. This fix should now move the line to about 45 degrees (if you scroll in a degree -45 - 45, the scroll is horizontal, 45 - 135 vertical). 

To test:
- Make sure the My Site improvements flag is turned on
- Go to a site with QS started
- Try scrolling horizontally and vertically to see if it feels better/more natural

PR submission checklist:
